react/BaseModal.tsx

```diff
@@ -20,6 +28,7 @@ interface Props
   disableEscapeKeyDown?: boolean
   children: React.ReactElement
   onBackdropClick?: (e: React.MouseEvent<HTMLDivElement>) => void
+  handles: CssHandlesTypes.CssHandles<typeof CSS_HANDLES>
```

Comment:
This component can be exported/imported if someone installs this app? If yes, this prop shouldn't be called `classes`? I don't have too much context

Reply:
Yes, it can, but it shouldn't 😅

Reply:
Yes, it can, but it shouldn't 😅